src/modules/Home/components/Home/Home.tsx

```diff
@@ -0,0 +1,324 @@
+						</p>
+						<p className={cn([styles.contentSectionText, styles.text])}>
+							Use any device that you have, even mobile
+							phone to create and test your images. CusDeb
+							provides new, unique way to interact with
+							linux in a cloud.
+						</p>
+						<button className={styles.readMoreButton}>Read More</button>
+					</Flex>
+
+				</Flex>
+			</Flex>
+		</div>
+
+		<div className={cn([styles.diagonal, styles.bitOSImages])}>
+			<Flex alignItems='center' className={styles.homeContainer}>
+				<Flex
+					className={styles.contentSection}
+					alignItems='center'
+				>
+					<Flex
+						className={styles.contentSectionCircle}
+						justifyContent='center'
+						alignItems='center'
+					>
+						<ChipIcon fill='#B1CFE8' className={styles.contentSectionCircleImg}/>
+					</Flex>
+
+					<Flex
+						className={styles.contentSectionInfo}
+						direction='column'
+						justifyContent='center'
+						alignItems='flex-start'
+					>
+						<p className={styles.contentSectionTitle}>
+							64 BIT OS IMAGES
+						</p>
+						<p className={cn([styles.contentSectionText, styles.text])}>
+							Unleash the power of your device with 64 bit built OS
+							images, that only CusDeb has for now.  This is going to
+							be a great experience for your device in case of performance.
+						</p>
+						<button className={styles.readMoreButton}>Read More</button>
+					</Flex>
+
+				</Flex>
+			</Flex>
+		</div>
+
+		<div className={cn([styles.diagonal, styles.diagonalRight, styles.formFactors])}>
+			<Flex alignItems='center' className={styles.homeContainer} justifyContent='flex-end'>
+				<Flex
+					className={cn([styles.contentSection, styles.contentSectionRight])}
+					alignItems='center'
+				>
+					<Flex
+						className={styles.contentSectionCircle}
+						justifyContent='center'
+						alignItems='center'
+					>
+						<img className={styles.contentSectionCircleImg} src={openedBookSVG} alt=''/>
+					</Flex>
+
+					<Flex
+						className={styles.contentSectionInfo}
+						direction='column'
+						justifyContent='center'
+					>
+						<p className={styles.contentSectionTitle}>
+							DETAILED DOCUMENTATION
+						</p>
+						<p className={cn([styles.contentSectionText, styles.text])}>
+							We have a great documentation which will help you to
+							learn more about how to build and use images on your
+							devices, and even more, we can learn you how to use
+							Linux and test it in our cloud terminal, this is gonna
+							be a lot of fun for you using CusDeb.
+						</p>
+						<button className={styles.readMoreButton}>Read More</button>
+					</Flex>
+
+				</Flex>
+			</Flex>
+		</div>
+
+		<Flex
+			className={cn([styles.prefooter, styles.diagonal])}
+			justifyContent='center'
+		>
+			<Flex
+				className={styles.prefooterContainer}
+				direction='column'
+				alignItems='center'
+			>
+				<p className={styles.prefooterTitle}>
+					Start using CusDeb today
+				</p>
```

Comment:
You have a lot of same mistakes in this file, please check it all

Reply:
All about i18n is done